### DISCUSSION TITLE
Discussions / support for labels

### DISCUSSION THREAD

**Discussion by IsaacSee** [community/community#3810] [Discussions] [answered]
Discussions can grow large quite fast, support for labels would greatly help handling scope, responsibility, state & priority!

**Comment by hliyan** [comment]
I agree. I find myself bucketing everything under "General". Labels (just as they are in issues) would be awesome.

**Comment by evi-liu** [comment, marked as answer]
Thanks for the feedback! Great news - labels are on the way :). Would your communities like to be included for private beta testing the feature once it's available?

**Reply by IsaacSee** [reply]
Awesome! Thank you @evi-liu. Yep you can include us in the beta!

**Reply by devashish-gaikwad** [reply]
Hello, I would also like to be included in this beta if possible!

Otherwise when do you think issue based search for mobile will be available?

**Reply by evi-liu** [reply]
Which repositories would you like to beta test Discussions labels on?

**Reply by IsaacSee** [reply]
Thanks @evi-liu, we can start with **[datahawk-technologies/datahawk-platform](https://github.com/datahawk-technologies/datahawk-platform/discussions)** :)

**Reply by evi-liu** [reply]
@IsaacSee ✅ enabled!

**Reply by andrewdmaclean** [reply]
@evi-liu if this is still possible it would be AMAZING!

**Reply by evi-liu** [reply]
@andrewdmaclean yup, which repository would you like to be flagged in?

**Reply by andrewdmaclean** [reply]
> 
> 
> @andrewdmaclean yup, which repository would you like to be flagged in?

@evi-liu it would be https://github.com/BattlesnakeOfficial/feedback! Thank you 😄

**Reply by evi-liu** [reply]
@andrewdmaclean ✅

**Reply by andrewdmaclean** [reply]
Thank you @evi-liu 🙏

**Reply by katelynnbarlowe** [reply]
I would use this a lot! Very helpful option.
https://github.com/deliciousbrains/spinupwp

**Reply by Kyaa-dost** [reply]
@evi-liu Yes, please! We would like the Labels option for our repository as well https://github.com/windows-toolkit/WindowsCommunityToolkit
CC: @michael-hawker

**Reply by JRoy** [reply]
If you could include [EssentialsX/Essentials](https://github.com/EssentialsX/Essentials) :)

**Reply by emmasmoot666** [reply]
#4013

**Reply by emmasmoot666** [reply]
#3536

**Reply by JRoy** [reply]
Hey @evi-liu just something I noticed: when converting an issue to a discussion, the labels of the issue are not converted over as well.

**Reply by evi-liu** [reply]
@JRoy Thanks for pointing this out! We're tracking this bug on our end and hope to have it fixed soon.
